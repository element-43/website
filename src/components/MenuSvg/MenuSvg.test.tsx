import { shallow, ShallowWrapper } from 'enzyme';
import * as React from 'react';

// Components.
import { IProps, MenuSvg } from './index';

interface IScope {
  props: IProps;
  wrapper: ShallowWrapper;
}

describe('src/components/MenuSvg', () => {
  let scope: IScope;

  beforeEach(() => {
    const props: IProps = {};

    scope = {
      props,
      wrapper: shallow(<MenuSvg {...props} />),
    };
  });

  describe('<MenuSvg /> snapshots', () => {
    it('should match the snapshot', () => {
      expect(scope.wrapper).toMatchSnapshot();
    });

    it('should match the snapshot with optional properties', () => {
      scope.props.color = '#000';
      scope.props.hoverColor = '#fff';
      scope.props.size = '50px';

      expect(shallow(<MenuSvg {...scope.props} />)).toMatchSnapshot();
    });
  });
});
