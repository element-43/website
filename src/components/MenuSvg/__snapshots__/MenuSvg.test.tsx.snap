// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/MenuSvg <MenuSvg /> snapshots should match the snapshot 1`] = `ShallowWrapper {}`;

exports[`src/components/MenuSvg <MenuSvg /> snapshots should match the snapshot with optional properties 1`] = `ShallowWrapper {}`;
